### ISSUE TITLE
AttributeError: 'Client' object has no attribute 'get_chat_history'

### ISSUE THREAD

**Issue by JustGuardian** [Dineshkarthik/telegram_media_downloader#288]
I just updated to version 2.0.0 and when I start the script I get this error
```
INFO     Started 12 HandlerTasks
Traceback (most recent call last):
  File "C:\Users\user\telegram_media_downloader-2.0.0\media_downloader.py", line 377, in <module>
    main()
  File "C:\Users\user\telegram_media_downloader-2.0.0\media_downloader.py", line 361, in main
    updated_config = asyncio.get_event_loop().run_until_complete(
  File "C:\Program Files\Python39\lib\asyncio\base_events.py", line 642, in run_until_complete
    return future.result()
  File "C:\Users\user\telegram_media_downloader-2.0.0\media_downloader.py", line 313, in begin_import
    messages_iter = client.get_chat_history(
AttributeError: 'Client' object has no attribute 'get_chat_history'
```

**Comment by Dineshkarthik** [comment]
@JustGuardian you need to  re-install the dependency packages. `pip install -r requirements.txt`
 I just now released V2.0.1 please update to the latest version as this fixes some bugs introduced in v2.0.0

**Comment by JustGuardian** [comment]
ok thanks for fast reply
